website/docs/kubernetes/deploying-a-container-to-kubernetes.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 10
+title: Deploying a container
+description: Deploying a container to Kubernetes 
+keywords: [podman desktop, podman, containers, migrating, kubernetes]
+tags: [migrating-to-kubernetes]
+---
+
+# Deploying a container to Kubernetes
+
+You can deploy a container to Kubernetes with Podman Desktop.
+
+#### Procedure
+
+1. Go to **Containers**.
+1. Click on a container name to go to the **Container Details** page.
+1. Click the <icon icon="fa-solid fa-rocket" size="lg" /> icon to generate a Kubernetes pod.
+1. In the **Deploy generated pod to Kubernetes** screen, choose your options:
+   1. Pod Name
+   2. Use Kubernetes Services to replace .`hostPort` exposure on containers by Services. It is the recommended way to expose ports, as a cluster policy may prevent to use `hostPort`.
+   2. Kubernetes Context
+   3. Namespace
+1. Click the **<icon icon="fa-solid fa-rocket" size="lg" /> Deploy** button.
+
+#### Verification
```

Comment:
it would be nice to expand on the capabilities. Meaning that if you have a service, it'll propose a link to open it directly from the UI

Reply:
I don't see that.

Reply:
I see the "Open Browser" icon in the *Container Details* page, but not in the *Pods Details* page. We should publish as is and add it later when the feature is ready.